_includes/header.html

```diff
@@ -1,8 +1,28 @@
 <header class="fx-row fx-row-between-xs fx-text-center">
-  <div class="fx-col fx-text-center"><a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
-      <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="40" height="40" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" /></a>
+  <div class="fx-col fx-text-center icon-container">
+    <a href="/{% if site.active_lang != site.default_lang %}{{ site.active_lang }}/{% endif %}">
+        <img alt="{{ site.data.general.nav.altJamulusIcon }}" width="45" height="45" src="{{ 'assets/img/jamulus-icon-2020.svg'| relative_url }}" />
+        <div class="icon">
+          <h1 class="icon-name">Jamulus</h1>
+          <div class="icon-tagline">{{ site.data.general.header.tagline }}</div>
+        </div>
+    </a> 
+  </div>
+        
+  </div>
+  <div class="fx-col-center-xs fx-txt-right headerlinks">
+    {% for item in site.data.headerlinks.header-links %}
+      <a 
+        href="{{ item.url }}" 
+        class="{% if item.url == "/download" %}download-header{% endif %}
+              {% if item.url == page.url %}current{% endif %}"
+              >
```

Comment:
This way of styling and adding parameters looks a bit odd to me. Are you sure this is correct?

Reply:
I'm open for any suggestions on styling.

Reply:
I'd just put the whole a element in one line.